content/docs/06-integrations/00-spectro-k8s-dashboard.md

```diff
@@ -0,0 +1,256 @@
+
+<br />
+
+<Tabs>
+
+<Tabs.TabPane tab="Basic OIDC Setup" key="Other Clouds">
+
+
+To enable RBAC OIDC authentication manually for clusters managed by most cloud providers, follow these steps: 
+
+<br />
+
+1. Copy ``oidc-`` configuration lines in the example and add them to the Kubernetes pack under the ``extraArgs`` parameter section. Enter your provider details in quotes. <br /><br />
+
+  ```
+  kubeadmconfig:
+    apiServer:
+      extraArgs:
+      oidc-issuer-url: "provider URL"
+      oidc-client-id: "client-id"
+      oidc-groups-claim: "groups"
+      oidc-username-claim: "email"
+  ```
+
+2. Under the ``clientConfig`` parameter section of Kubernetes pack, uncomment the ``oidc-`` configuration lines, and enter your provider details in quotes. Enter the same provider URL and client-id in the ``extraArgs`` and ``clientConfig`` parameter sections. <br /><br />
+
+  ```
+    clientConfig:
+      oidc-issuer-url: "provider URL"
+      oidc-client-id: "client-id"
+      oidc-client-secret: client-secret-value
+      oidc-extra-scope: profile,email,openid
+    ```
+
+<br />
+
+
+</Tabs.TabPane>
+
+<Tabs.TabPane tab="AWS EKS" key="AWS EKS">
+
+To enable RBAC OIDC authentication manually for EKS clusters, follow these steps: 
+
+<br />
+
+1. In the Kubernetes pack, uncomment the lines in the ``oidcIdentityProvider`` parameter section, and enter your third-party IDP details.
+<br />
+
+  ```
+    oidcIdentityProvider:
+      identityProviderConfigName: 'Spectro-docs'     # The name of the OIDC provider configuration
+      issuerUrl: 'issuer-url'       # The URL of the OpenID identity provider
+      clientId: 'user-client-id-from-Palette'           # The ID for the client application that makes authentication requests to the OpenID identity provider
+      usernameClaim: "email"                     # The JSON Web Token (JWT) claim to use as the username
+      usernamePrefix: "-"                        # The prefix that is prepended to username claims to prevent clashes with existing names
+      groupsClaim: "groups"                      # The JWT claim that the provider uses to return your groups
+      groupsPrefix: ""                          # The prefix that is prepended to group claims to prevent clashes with existing names
+      requiredClaims:                            # The key value pairs that describe required claims in the identity token
+  ```
+
+<br />
+
+
+2. Under the ``clientConfig`` parameter section of Kubernetes pack, uncomment the ``oidc-`` configuration lines. <br /><br />
+
+  ```
+  clientConfig:
+    oidc-issuer-url: "{{ .spectro.pack.kubernetes-eks.managedControlPlane.oidcIdentityProvider.issuerUrl }}"
+    oidc-client-id: "{{ .spectro.pack.kubernetes-eks.managedControlPlane.oidcIdentityProvider.clientId }}"
+    oidc-client-secret: 1gsranjjmdgahm10j8r6m47ejokm9kafvcbhi3d48jlc3rfpprhv
+    oidc-extra-scope: profile,email
+  ``` 
+
+</Tabs.TabPane>
+
+<Tabs.TabPane tab="Azure AKS" key="Azure AKS">
+ 
```

Comment:
@MGamya @arvind-sys 
Please review this AKS section carefully. Are all the necessary steps included? What is the command in step 5 all about? What does it do?

Reply:
Question no longer applies. Azure AKS tab is removed.